### ISSUE TITLE
Preset option with "conventionalcommits" doesn't seem to work 

### ISSUE THREAD

**Issue by JellyBellyDev** [TriPSs/conventional-changelog-action#246]
My config:
```yaml
    steps:
      - name: Checkout code
        uses: actions/checkout@v2

      - name: Conventional Changelog Action
        id: changelog
        uses: TriPSs/conventional-changelog-action@v5
        with:
          preset: conventionalcommits
          github-token: ${{ secrets.GITHUB_TOKEN }}
          git-user-name: ${{ github.actor }}
          git-user-email: ${{ github.actor }}@users.noreply.github.com
          tag-prefix: 'v'
```
and in action log i see:
```
Error: Error: Unable to load the "conventionalcommits" preset. Please make sure it's installed.
```
The possible values are not these: https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-changelog/README.md#preset ?

Thanks

**Comment by TriPSs** [comment]
When using presets other than `angular` you need to make sure they are [installed](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-conventionalcommits#indirect-usage-as-preset).

**Comment by JellyBellyDev** [comment]
@TriPSs thanks for the quick reply. Where e how install it? My software is not a node application.
Is possible ti install it in the github actions?

**Comment by TriPSs** [comment]
Aah, yea you would need to install it with node, but since this request already came multiple times let me bundle this one with the action.

**Comment by JellyBellyDev** [comment]
Ok! Thank you! 
I'm waiting for news then! 💪

**Comment by TriPSs** [comment]
It's released, versions: `v5` and `v5.1.0`

**Comment by JellyBellyDev** [comment]
AWESOME!!! Works!!! ❤️
